analysis.R

```diff
@@ -0,0 +1,122 @@
+
+
+#Figuring out which markers cause the disease
+total.cases.X <- 0
+for (i in 1:nrow(countryX.df)){
+  if(check_infected(i,countryX.df) == T){ 
+    total.cases.X <- total.cases.X +1
+    }
+}
+total.cases.Y <- 0
+for (i in 1:nrow(countryY.df)){
+  if(check_infected(i,countryY.df) == T){ 
+    total.cases.Y <- total.cases.Y +1
+  }}
+
+x.markers.pct <- numeric(10)
+y.markers.pct <- numeric(10)
+x.markers.pct[1] <- sum(countryX.df[,"marker01"] == "1") / total.cases.X
+y.markers.pct[1] <- sum(countryY.df[,"marker01"] == "1") / total.cases.Y
+x.markers.pct[2] <- sum(countryX.df[,"marker02"] == "1") / total.cases.X
+y.markers.pct[2] <- sum(countryY.df[,"marker02"] == "1") / total.cases.Y
+x.markers.pct[3] <- sum(countryX.df[,"marker03"] == "1") / total.cases.X
+y.markers.pct[3] <- sum(countryY.df[,"marker03"] == "1") / total.cases.Y
+x.markers.pct[4] <- sum(countryX.df[,"marker04"] == "1") / total.cases.X
+y.markers.pct[4] <- sum(countryY.df[,"marker04"] == "1") / total.cases.Y
+x.markers.pct[5] <- sum(countryX.df[,"marker05"] == "1") / total.cases.X
+y.markers.pct[5] <- sum(countryY.df[,"marker05"] == "1") / total.cases.Y
+x.markers.pct[6] <- sum(countryX.df[,"marker06"] == "1") / total.cases.X
+y.markers.pct[6] <- sum(countryY.df[,"marker06"] == "1") / total.cases.Y
+x.markers.pct[7] <- sum(countryX.df[,"marker07"] == "1") / total.cases.X
+y.markers.pct[7] <- sum(countryY.df[,"marker07"] == "1") / total.cases.Y
+x.markers.pct[8] <- sum(countryX.df[,"marker08"] == "1") / total.cases.X
+y.markers.pct[8] <- sum(countryY.df[,"marker08"] == "1") / total.cases.Y
+x.markers.pct[9] <- sum(countryX.df[,"marker09"] == "1") / total.cases.X
+y.markers.pct[9] <- sum(countryY.df[,"marker09"] == "1") / total.cases.Y
+x.markers.pct[10] <- sum(countryX.df[,"marker10"] == "1") / total.cases.X
+y.markers.pct[10] <- sum(countryY.df[,"marker10"] == "1") / total.cases.Y
+markers.list <- as.character(seq(1,10))
+markers.sum.table <- matrix(ncol = 3,nrow=20)
+colnames(markers.sum.table) <- c("country","marker","freq")
+markers.sum.table <- as.data.frame(markers.sum.table)
+for (i in 1:10){
+  tempvect01 <- c("X",as.character(i),x.markers.pct[i])
+  markers.sum.table[i,] <- tempvect01
+  tempvect02 <- c("Y",as.character(i),y.markers.pct[i])
+  markers.sum.table[10+i,] <- tempvect02
+}
+#plotting the markers that cause disease
+# Note how the distribution of markers for country X is skewed towards Markers 1-5,
+# Whereas Country Y has a relatively even distribution. This means that a vaccine targeted
+# At patients in country Y might be based upon solving a problem caused by a micro satellite the patient in 
+# Country X might not have present. Thus, vaccines developed by country Y MIGHT work for 
+# Country X, but it is not certain.
+ggplot(data=markers.sum.table,aes(x=marker,y=freq,fill=country)) + 
+  geom_bar(stat="identity",position = "dodge",alpha=.75) + 
+  ggtitle("Relative Frequency of MicroSatellites")
+  
+
+
+
```

Comment:
Good commenting and efficient code that uses coding concepts that we covered in class – 3.25 points (max is 4)